src/script/zcash_script.h

```diff
@@ -99,6 +119,32 @@ EXPORT_SYMBOL int zcash_script_verify(
     uint32_t consensusBranchId,
     zcash_script_error* err);
 
+/// Returns 1 if the input nIn of the serialized transaction pointed to by
+/// txTo correctly spends the scriptPubKey pointed to by scriptPubKey under
+/// the additional constraints specified by flags. Must be used for V5 transactions;
+/// may also be used for previous versions.
+///
+/// allPrevOutputs must point to the concatenation of all outputs from previous
+/// transactions that are spent by the inputs of the given transaction.
+/// The outputs must be encoded as specified by Bitcoin (value followed by
+/// pk_script, including a leading CompactSize).
+///
+/// If not NULL, err will contain an error/success code for the operation.
+/// Note that script verification failure is indicated by err being set to
+/// zcash_script_ERR_OK and a return value of 0.
+EXPORT_SYMBOL int zcash_script_verify_v5(
+    const unsigned char* scriptPubKey,
+    unsigned int scriptPubKeyLen,
+    int64_t amount,
```

Comment:
Both of these are unnecessary in this API; we can always obtain them from `allPrevOutputs[nIn]`. This will also be correct for pre-v5 transactions: I double-checked, and `scriptPubKey` will always be interpreted as the actual `scriptPubKey` here, never `redeemScript` for P2SH (that component of the logic is handled inside this function, not by the caller).
```suggestion
```

Reply:
That's true! Removed in b459f197f